ibis/backends/bigquery/compiler.py

```diff
@@ -338,7 +340,11 @@ def _formatter(translator, expr):
 }
 
 
-_operation_registry = impala_compiler._operation_registry.copy()
+# _operation_registry = impala_compiler._operation_registry.copy()
```

Comment:
can be in a followon; if we are importing from / to another module then these should be non-private (no leading _)

Reply:
noted. i had left the original impala updates as is. but will update these on next batch.